velox/functions/lib/string/tests/StringImplTest.cpp

```diff
@@ -248,6 +249,41 @@ TEST_F(StringImplTest, charToCodePoint) {
   testExpectDeath("");
 }
 
+TEST_F(StringImplTest, stringToCodePoint) {
+  auto testValidInput = [](const std::string& charString,
+                           const std::vector<int32_t> expectedCodePoint,
+                           const bool isAscii = true) {
+    std::vector<int32_t> res;
+    if (isAscii) {
+      res = stringToCodePoint</*isAscii*/ true>(StringView(charString));
+    } else {
+      res = stringToCodePoint</*isAscii*/ false>(StringView(charString));
+    }
+    for (int i = 0; i < res.size(); i++) {
+      ASSERT_EQ(res.at(i), expectedCodePoint.at(i));
+    }
+  };
+  testValidInput("", {});
+  testValidInput("h", {0x0068});
+  testValidInput("hello", {0x0068, 0x0065, 0x006C, 0x006C, 0x006F});
+  testValidInput("hïllo", {0x0068, 0x00EF, 0x006C, 0x006C, 0x006F}, false);
```

Comment:
For readability, rename testValidInput to testAscii and change the implementation to call stringToCodePoint<true> and stringToCodePoint<false> and verify both results against provided expected result.

Then, add another lambda for non-ASCII and use that to test UTF-8 strings. Consider spitting the test into 2: one for ascii and another for non-ascii strings.